core/platform/datastore/gae_datastore_services.py

```diff
@@ -31,7 +31,17 @@
 from google.appengine.datastore import datastore_stub_util
 from google.appengine.ext import ndb
 
+from typing import ( # isort:skip # pylint: disable=unused-import
+    Any, Callable, Iterator, List, Optional, Sequence,
+    Text, Tuple, TypeVar)
+
+MYPY = False
+if MYPY:
+    from mypy_imports import * # pragma: no cover # pylint: disable=import-only-modules,wildcard-import,unused-wildcard-import
```

Comment:
I've commented in a previous PR, multiple times, on not using *.

Please fix all the issues here that arise from the previous code review, and then you can assign reviewers. You should check the Files Changed tab to ensure that everything is correct before assigning reviewers (i.e., do a self-review).

Reply:
Actually I am waiting for PR1 to get completed to merge it to this PR. That's why I wanted to get files of some specific folders in this PR to be checked as this is a stacked PR. See the files [here](https://github.com/oppia/oppia/pull/13524#issuecomment-890983198).

Thanks!!